app/scripts/modules/ecs/src/serverGroup/configure/wizard/horizontalScaling/horizontalScaling.component.html

```diff
@@ -38,63 +38,102 @@
       <help-field key="ecs.capacityProviderStrategy"></help-field>
     </div>
     <div>
+      <div class="radio">
```

Comment:
I thought we discussed moving these components to using React rather than keeping them as Angular components @allisaurus correct me if I'm mistaken

Reply:
@paragbhingre you can use https://github.com/spinnaker/deck/pull/8699 as reference (migrated the component to react in one pr, then implement your feature on top in another PR)

Reply:
I had already started working on deck (using angular) when we last decided to push the code before the release. I had a conversation with @allisaurus and she mentioned that first of all we need to change the existing code into react and then on top of it we can make our changes for CP which was too much work before the release. And after release, I had almost completed my work in angular just left with the functional tests. That is why this code is in angular and not in react. But if we think that can be our priority in this release then I am happy to do that.

Reply:
My opinion was/is that migrating the entire `horizontalScaling` component to react was beyond the scope of this PR, but that the _dedicated capacity provider inputs_ should be housed in their own react component which `horizontalScaling` component could then consume. In that case, the contents of the `<div>` on line 35 should look something like:

```
<div class="form-group" ng-if="$ctrl.capacityProviderState.useCapacityProviders">
    <ecs-capacity-provider-selector
        command="$ctrl.command"
        capacity-provider-state="$ctrl.capacityProviderState">
    </ecs-capacity-provider-selector>
</div>
```  
...with the entirety of the selection/data population logic in the new react component.

We can still take that approach, however since we're a ways out from a release boundary now I'm also cool with migrating `horizontalScaling` to react (separate PR) and then rebasing this capacity provider enhancement on top of it. I'm fine with whichever is easiest given the changes needed.

Reply:
Makes sense @allisaurus. I'm alright with leaving the migration of the whole component to a potential task after the fact (as that's a substantial amount of work). That being said, I agreed with the above comment that any new components/functionality that we add should be using react rather than angular (as that's the approach that we're moving towards)